bindata/manifests/machine-config/files/switchedev.conf.yaml

```diff
@@ -0,0 +1,6 @@
+mode: 0755
```

Comment:
why this file cannot be written by sriov config daemon? is it due to the file mode 0755? (we have a node state checkpoint file written by this daemon)
nit: there is an extra `e` in the file name.